stew/ptrops.nim

```diff
@@ -34,14 +34,8 @@ template offset*[T](p: ptr T, count: int): ptr T =
 
   # Actual behavior is wrapping, but this may be revised in the future to enable
   # better optimizations.
-
-  # We turn off checking here - too large counts is UB
-  {.push checks: off.}
-  let
-    bytes = count * sizeof(T)
-    res = cast[ptr T](offset(cast[pointer](p), bytes))
-  {.pop.}
-  res
+  let bytes = count * sizeof(T)
```

Comment:
this is the overflow that the checks off was supposed to prevent - a large count of large items should simply wrap, to maintain previous behavior